interfaces/acados_template/acados_template/c_templates_tera/acados_solver.in.c

```diff
@@ -563,15 +559,15 @@ int acados_create() {
         ocp_nlp_dims_set_constraints(nlp_config, nlp_dims, i, "nh", &nh[i]);
     }
 
-    {% if dims.npd > 0 %}
+    {%- if dims.npd > 0 %}
     for (int i = 0; i < N; i++) 
         ocp_nlp_dims_set_constraints(nlp_config, nlp_dims, i, "np", &npd[i]);
-    {% endif %}
-    {% if dims.npd_e > 0 %}
+    {%- endif %}
```

Comment:
why did you add all the -s? Does it improve indentation?

Reply:
It is used to get less empty lines in case that this part is not rendered.